pubspec.lock

```diff
@@ -7,14 +7,14 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.5.0-nullsafety"
+    version: "2.4.2"
```

Comment:
Only just noticed the `nullsafety` diff.. are you using a tech preview?
  https://dart.dev/null-safety

Sounds groovy, do we want to use that everywhere even if they say not to use it in production?

Reply:
yes. using preview version. Should work without tho